tests/k8s/run-tests.bash

```diff
@@ -65,10 +65,11 @@ function run_tests(){
     # Set up cilium-lb-ds and cilium-ds
     deploy_cilium ${k8s_version}
 
+
     # Run non IP version specific tests
-    vmssh ${node2} 'set -e; for test in /home/vagrant/go/src/github.com/cilium/cilium/tests/k8s/tests/*.sh; do $test; done'
```

Comment:
What's the reason for this?

Reply:
This allows us to save the output from each test into the folder that is packaged up and saved as an artifact of each build.

Reply:
Ah got it, it was difficult to read it with that path length